include/script_api/macros.h

```diff
@@ -16,7 +16,9 @@
 
 /// Decimal constant.
 /// Despite the name, "floats" are actually stored as fixed-point values.
-#define EVT_FLOAT(DOUBLE) (((Bytecode)(DOUBLE * 1024.0f) + -230000000))
+
+#define ROUND(x) ((x)>=0?(long)((x) + 0.9):(long)(x))
```

Comment:
Preferably would use f64 instead of long imo and explain why this macro is necessary. Also should probably be called `_EVT_FLOAT_ROUND` or something to be clear it isn't meant to be used elsewhere